docs/source/prototype.pipelines.rst

```diff
@@ -24,3 +24,27 @@ EMFORMER_RNNT_BASE_TEDLIUM3
 
    .. autodata:: EMFORMER_RNNT_BASE_TEDLIUM3
       :no-value:
+
+
+HiFiGAN Generator
```

Comment:
Although the model and factory functions are named by Generator, I think we can make the bundle title to be "HiFiGAN Vocoder" so that users can better understand what is the model used for.

Reply:
Sure, renamed the bundle to "HiFiGAN Vocoder"